libs/remixd/src/services/slitherClient.ts

```diff
@@ -0,0 +1,110 @@
+      obj.description = e.description
+      obj.title = e.check
+      obj.confidence = e.confidence
+      obj.severity = e.impact
+      obj.sourceMap = e.elements.map((element) => {
+        delete element.source_mapping.filename_used
+        delete element.source_mapping.filename_absolute
+        return element
+      })
+      standardReport.push(obj)
+    }
+    return standardReport
+  }
+
+  analyse (filePath: string, compilerConfig: Record<string, any>) {
+    return new Promise((resolve, reject) => {
+      if (this.readOnly) {
+        const errMsg: string = '[Slither Analysis]: Cannot analyse in read-only mode'
+        return reject(new Error(errMsg))
+      }
+      const options = { cwd: this.currentSharedFolder, shell: true }
+      const { currentVersion, optimize, evmVersion } = compilerConfig
+      if (currentVersion && currentVersion.includes('+commit')) {
+        // Get compiler version with commit id e.g: 0.8.2+commit.661d110
+        const versionString: string = currentVersion.substring(0, currentVersion.indexOf('+commit') + 16)
+        console.log('\x1b[32m%s\x1b[0m', `[Slither Analysis]: Compiler version is ${versionString}`)
+        // Check solc current installed version
+        const solcOutput: Buffer = execSync('solc --version', options)
+        if (!solcOutput.toString().includes(versionString)) {
+          console.log('\x1b[32m%s\x1b[0m', '[Slither Analysis]: Compiler version is different from installed solc version')
+          // Get compiler version without commit id e.g: 0.8.2
+          const version: string = versionString.substring(0, versionString.indexOf('+commit'))
+          // List solc versions installed using solc-select
+          const solcSelectInstalledVersions: Buffer = execSync('solc-select versions', options)
+          // Check if required version is already installed
+          if (!solcSelectInstalledVersions.toString().includes(version)) {
+            console.log('\x1b[32m%s\x1b[0m', `[Slither Analysis]: Installing ${version} using solc-select`)
+            // Install required version
+            execSync(`solc-select install ${version}`, options)
+          }
+          console.log('\x1b[32m%s\x1b[0m', `[Slither Analysis]: Setting ${version} as current solc version using solc-select`)
+          // Set solc current version as required version
+          execSync(`solc-select use ${version}`, options)
+        } else console.log('\x1b[32m%s\x1b[0m', '[Slither Analysis]: Compiler version is same as installed solc version')
+      }
+      const outputFile: string = 'remix-slitherReport_' + Date.now() + '.json'
+      const optimizeOption: string = optimize ? '--optimize ' : ''
+      const evmOption: string = evmVersion ? `--evm-version ${evmVersion}` : ''
+      const solcArgs: string = optimizeOption || evmOption ? `--solc-args '${optimizeOption}${evmOption}'` : ''
+      const cmd: string = `slither ${filePath} ${solcArgs} --json ${outputFile}`
```

Comment:
Is there any way to tell the compiler how to resolve import?

running this npm import fails with:
```
crytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Source "@openzeppelin/contracts@4.1.0/token/ERC20/ERC20.sol" not found: File not found.
 --> remix-project/apps/remix-ide/contracts/contract1.sol:4:1:
  |
4 | import "@openzeppelin/contracts@4.1.0/token/ERC20/ERC20.sol";
```